packages/web-components/stencil.config.ts

```diff
@@ -1,12 +1,16 @@
 import { Config } from '@stencil/core';
 import { postcss } from '@stencil/postcss';
+import { sass } from '@stencil/sass';
 import url from 'postcss-url';
 import { reactOutputTarget } from '@stencil/react-output-target';
 
 export const config: Config = {
   namespace: 'component-library',
   globalStyle: 'src/global/main.css',
   plugins: [
+    sass({ 
+      includePaths: ["../../node_modules/@uswds/uswds/packages/"],
+    }),
```

Comment:
Because of the way things changed along the way for this component, I ended up adding the new NPM packages to this PR (@stencil/sass & @uswds/uswds) but if we feel like this and the following related files should be its own PR, I can pull it out.